### ISSUE TITLE
can i have more than 1 worker for handling sockets with uvloop?

### ISSUE THREAD

**Issue by catroot** [quantmind/pulsar#262]
'python test_pulsar.py --log-level warning --io uv --concurrency multi --workers 2'  raises error
```
..........
  File "uvloop/loop.pyx", line 1364, in uvloop.loop.Loop.create_server (uvloop/loop.c:26065)
  File "uvloop/handles/streamserver.pyx", line 43, in uvloop.loop.UVStreamServer.listen (uvloop/loop.c:72364)
  File "uvloop/handles/streamserver.pyx", line 80, in uvloop.loop.UVStreamServer._fatal_error (uvloop/loop.c:72849)
OSError: [Errno 88] Socket operation on non-socket
```

'python test_pulsar.py --log-level warning --io uv --workers 2' raises
```
  File "/usr/lib/python3.5/multiprocessing/reduction.py", line 50, in dumps
    cls(buf, protocol).dump(obj)
  File "/usr/lib/python3.5/multiprocessing/reduction.py", line 235, in _reduce_socket
    df = DupFd(s.fileno())
  File "/usr/lib/python3.5/multiprocessing/reduction.py", line 190, in DupFd
    return resource_sharer.DupFd(fd)
  File "/usr/lib/python3.5/multiprocessing/resource_sharer.py", line 48, in __init__
    new_fd = os.dup(fd)
OSError: [Errno 9] Bad file descriptor
```

`python test_pulsar.py --log-level warning --io uv --workers 1` works fine

Is these means that i can`t run more than 1 worker for handling sockets with uvloop?

The content of the test_pulsar.py file is the hello world app from the manual:
```python
from pulsar.apps import wsgi

def hello(environ, start_response):
    data = b'Hello World!\n'
    response_headers = [
        ('Content-type','text/plain'),
        ('Content-Length', str(len(data)))
    ]
    start_response('200 OK', response_headers)
    return [data]


if __name__ == '__main__':
    wsgi.WSGIServer(callable=hello).start()
```

**Comment by lsbardel** [comment]
Hi, this should not happen, but there are some issues with ``uvloop`` at the moment (see #258).
You don't need to pass `--concurrency multi`, by default pulsar uses subprocess concurrency.

I'm waiting on python 3.6 release to finalise the fix, many changes have been made in the new python asyncio module.

**Comment by lsbardel** [comment]
Issue opened MagicStack/uvloop#65

**Comment by lsbardel** [comment]
This has been fixed in the master branch.
The fix will be included in the 1.6.4 release. Thanks for reporting.
